pysteps/verification/salscores.py

```diff
@@ -155,11 +155,11 @@ def sal_structure(
     observation_objects = _sal_detect_objects(
         observation, thr_factor, thr_quantile, tstorm_kwargs
     )
-    prediction_volume = _sal_scaled_volume(prediction_objects).sum()
-    observation_volume = _sal_scaled_volume(observation_objects).sum()
+    prediction_volume = sal_scaled_volume(prediction_objects)
+    observation_volume = sal_scaled_volume(observation_objects)
     nom = prediction_volume - observation_volume
     denom = prediction_volume + observation_volume
-    return np.divide(nom, (0.5 * denom))
+    return nom / (0.5 * denom)
```

Comment:
I changed this to `np.divide()` in #327 because we want this function to raise an error instead of returning `Inf`, any particular reason to change it back?

Reply:
Hi @dnerini, thank you.
You are right. I've applied black and edited these lines .
Merry Christmas :)